sql/src/main/java/io/crate/planner/Planner.java

```diff
@@ -170,6 +152,14 @@ public Plan planSingleRowSubselect(AnalyzedStatement statement) {
                 planner, clusterService, subJobId, consumingPlanner, normalizer, transactionContext, 2, 2));
         }
 
+        QuerySpec applySoftLimit(QuerySpec querySpec) {
```

Comment:
Imho returning the same query spec that we pass as argument is a bit confusing...